File: app/src/main/java/com/neome/java/api/ent/entDrawer/msg/MsgEntVariableUpdate.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.entDrawer.msg;

import com.google.gson.JsonElement;
import com.neome.java.api.meta.base.Types.MetaIdVar;
import com.neome.java.api.nucleus.base.msg.Msg;

import org.jetbrains.annotations.Nullable;

import java.util.Map;

@SuppressWarnings("unused")
public class MsgEntVariableUpdate extends Msg
{
  @Nullable
  public Map<MetaIdVar, JsonElement> variableObjectMap;
}

File: app/src/main/java/com/neome/api/ent/entDrawer/msg/MsgEntVariableUpdate.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entDrawer.msg

import kotlinx.serialization.json.JsonElement
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.nucleus.base.msg.Msg

interface MsgEntVariableUpdate : Msg
{
  val variableObjectMap: Map<MetaIdVar, JsonElement>?
}